src/App.css

```diff
@@ -32,6 +32,7 @@
   from {
     transform: rotate(0deg);
   }
+
   to {
     transform: rotate(360deg);
   }
```

Comment:
# Optional
- **Design: I would advise you to follow the design guidelines including Typography( Font, size), Layout (Composition and space between elements). It is part of the project requirements**
- **See your design:**
![Capture](https://user-images.githubusercontent.com/63972670/219782459-c3e754ca-b429-4ccf-9b7e-f35716f2cb5d.PNG)
- Your design: 
![Capture1](https://user-images.githubusercontent.com/63972670/219782676-ac8e7932-002c-46a7-848b-62f8bfde4962.PNG)
-
- **Original Design:**
![1](https://user-images.githubusercontent.com/63972670/219782961-cefe543a-672d-4927-af71-32669ad138ac.PNG)

Reply:
@OLIPLICHE Thank you for all the suggestions especially the optional ones I will definitely work on them to improve my project